src/python/pants/core/util_rules/external_tool.py

```diff
@@ -119,6 +120,34 @@ def register_options(cls, register):
             advanced=True,
             help=help_str,
         )
+        register(
+            "--url-template",
+            type=str,
+            default=cls.default_url_template,
+            advanced=True,
+            help=(
+                "URL to download the tool, either as a single binary file or a compressed file "
+                "(e.g. zip file). You can change this to point to your own hosted file, e.g. to "
+                "work with proxies. Use {version} to have the value from --version substituted, "
+                "and {platform} to have a value from --url-platform-mapping substituted in, "
```

Comment:
Might want to add the ` around the {version} part and {platform} part to express that the values should be used exactly.